docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,374 @@
+
+ramips-mt7628 [#newtarget]_ [#noibss]_
+======================================
+
+* VoCore
+
+  - VoCore 2
+
+ramips-rt305x [#newtarget]_ [#noibss]_
+======================================
+
+* A5
+
+  - V11
+
+* D-Link
+
+  - DIR615 (D1, D2, D3, D4, H1)
+
+* VoCore
+
+  - VoCore (8MB, 16MB)
+
+sunxi [#newtarget]_
+===================
+
+* LeMaker
+
+  - Lamobo r1
+
+
+.. [#newtarget]
+  New target
+
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+New features
+************
+
+Multidomain support
+===================
+
+When mesh networks grow too large, it is often necessary to split them into
+multiple independent mesh domains to allow the meshes to work with acceptable
+performance. Formerly, the only way to achieve this with Gluon was to build
+a separate set of firmware images for each domain.
+
+With Gluon 2018.1, multidomain firmwares can be used to achieve the same,
+using only a single site configuration, with only short snippets of domain-specific
+configuration. The feature is explained in detail in :doc:`../features/multidomain`.
+
+Wired mesh encapsulation
+========================
+
+Gluon now supports encapsulating wired mesh traffic (Mesh on LAN/WAN) in
+`VXLAN <https://en.wikipedia.org/wiki/Virtual_Extensible_LAN>`_.
+See :doc:`../features/wired-mesh` for details on this feature.
+
+Router advertisement filtering
+==============================
+
+Similar to the builtin batman-adv gateway feature for IPv4, the *gluon-radv-filterd* package
+(*radv-filterd* feature flag) allows to filter IPv6 router advertisements from the mesh
+so that only the RAs with the best routing metric (TQ) reach the clients, ensuring that
+the "best" gateway is chosen as the IPv6 default route.
+
+At the moment, this feature only filters RAs forwarded to clients; the RAs handled on
+the nodes themselves will be unfiltered, so the nodes will still use arbitrary default
+gateways.
+
+IGMP/MLD segmentation
+=====================
+
+The IGMP/MLD segmentation feature previously provided by the *gluon-ebtables-segment-mld*
+package has been extended and moved into the Gluon core; it does not exist as a separate package
+anymore.
+
+Filtering IGMP/MLD queries directed towards the mesh ensures that each node becomes the multicast querier
+for its own clients (unless there are other multicast-aware switches connected to the node), rather
+than electing a single, basically arbitrary node in the mesh to become the querier. Overall,
+this should significantly improve the reliablity of multicast in the mesh. This is especially
+important for IPv6, as the IPv6 Neighbour Discovery Protocol (NDP) is based on local multicast.
+
+See also the documentation of the :ref:`site.conf mesh section <user-site-mesh>`.
+
+gluon-ebtables-limit-arp
+========================
+
+The *gluon-ebtables-limit-arp* (*ebtables-limit-arp* feature flag) package adds filters to limit the
```

Comment:
to limit the rate at which clients are allowed to send ARP requests into the mesh.